mythril/laser/ethereum/call.py

```diff
@@ -179,16 +180,26 @@ def get_call_data(
     )
 
     try:
-        calldata_from_mem = state.memory[
-            util.get_concrete_int(memory_start) : util.get_concrete_int(
-                memory_start + memory_size
+        uses_entire_calldata = False
+        try:
+            uses_entire_calldata = simplify(
```

Comment:
is the "try" statement in line 182 needed to be covered for these lines? As i think TypeError should be caught from lines 194:197 and not from these lines right?